### PR TITLE
#3813 Reset pipeline state when switching between different pipelines

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3813 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/PipelineHooks.js [hunk 1]
```diff
@@ -90,7 +90,10 @@ export const usePipelineState = () => {
           ...state,
           isMetricsOn: !state.isMetricsOn,
         };
-
+      case 'resetPipeline':
+        return {
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/PipelineHooks.js]:
```suggestion
        return state;
```

File: ohara-manager/client/src/components/Pipeline/Pipeline.js [hunk 2]
```diff
@@ -184,6 +189,13 @@ const Pipeline = React.forwardRef((props, ref) => {
 
   const { isToolboxOpen, toolboxExpanded, toolboxKey } = pipelineState;
 
+  //We need wait all pipelineApi is ready ,next render pipeline.
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Pipeline.js]:
```suggestion
  //  If paper API is not ready, let's reset the pipeline state and re-render again
```

File: ohara-manager/client/src/components/Pipeline/PipelineHooks.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/Pipeline/PipelineHooks.js]: ok nice~

File: ohara-manager/client/src/components/Pipeline/Pipeline.js (hunk 2 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/Pipeline/Pipeline.js]: oh!!! thanks!

**Pull request merged**